testing/src/main/java/net/consensys/linea/testing/ToyExecutionEnvironmentV2.java

```diff
@@ -147,7 +147,7 @@ private MainnetTransactionProcessor getMainnetTransactionProcessor(EVM evm) {
         new MessageCallProcessor(evm, precompileContractRegistry);
 
     final ContractCreationProcessor contractCreationProcessor =
-        new ContractCreationProcessor(evm, false, List.of(), 0);
+        new ContractCreationProcessor(evm, false, List.of(), 1);
```

Comment:
What's the impact ?

Reply:
It was first noticed by @davepearce. It means that accounts created start out with a nonce of 1 (as in the EVM since some old hard fork)